locations/spiders/dsv_locker_za.py

```diff
@@ -0,0 +1,30 @@
+from scrapy import Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class DSVLockerZASpider(Spider):
+    name = "dsv_locker_za"
+    item_attributes = {"brand": "DSV Locker", "brand_wikidata": "Q1155771"}
+    allowed_domains = ["dsv-clientzone-map.afrigis.co.za"]
+    start_urls = ["https://dsv-clientzone-map.afrigis.co.za/proxy/api/wfs/?layer=locker"]
+
+    def parse(self, response):
+        for data in response.json()["result"]["features"]:
+            data = data["properties"]
+            item = DictParser.parse(data)
```

Comment:
The source data has `data["location_code"]` which is perhaps a better choice for `item["ref"]` than leaving things to default where you get `data["id"]` instead. It looks like `id` is just local to the store finder, whereas `location_code` is the brand's internal site code that is used across systems.